CONTRIBUTING.md

```diff
@@ -30,34 +30,73 @@ Follow the steps below when submitting your code.
 
 1. Search the [list of issues](https://github.com/DevExpress/testcafe-hammerhead/issues) to see if there is an issue for the bug or feature you are going to work on or create a new one.
 
-2. If you are going to address an existing issue, check the comment thread to make sure that nobody is working on it at the moment.
+2. To address an already described issue, check the comment thread to make sure that nobody is working on it at the moment. Leave a comment saying that you are willing to fix this issue, and include details on how you plan to do this. Core team members may need to discuss the details of the proposed fix with you. After the green light from them,
+leave a comment saying that you started your work on this issue.
```

Comment:
Just out of curiousity, why is the last part of line 33 on this line instead?

Reply:
That's a mistake. Good that you've noticed, thanks.